x-pack/test/apm_api_integration/tests/event_metadata/event_metadata.spec.ts

```diff
@@ -32,12 +33,18 @@ export default function ApiTest({ getService }: FtrProviderContext) {
       },
     });
 
-    return response.hits.hits[0]._source![processorEvent].id;
+    const doc = response.hits.hits[0]._source!;
+
+    return {
+      // @ts-expect-error
+      id: doc[processorEvent].id as string,
+      timestamp: doc['@timestamp'],
+    };
   }
 
   registry.when('Event metadata', { config: 'basic', archives: ['apm_8.0.0'] }, () => {
```

Comment:
nit: We will have to start adhering to the new API guidelines, which Dario is creating where if someone works on an API test which use Archives, then we should migrate it to Synthtrace or Index using ES. This way we will soon get rid of all Archives without spending dedicate time.

Reply:
That's a good point. I'll convert this asap